### PR TITLE
Gut stale roadmaps. Move useful content elsewhere.

### PR DESCRIPTION
**Pull request by bgrant0607**
cc @thockin

### CODE REVIEW

**Comment by bgrant0607** [comment]
cc @davidopp

File: docs/admin/cluster-large.md [hunk 1]
```diff
@@ -35,10 +35,12 @@ Documentation for other releases can be found at
 
 ## Support
 
-At v1.0, Kubernetes supports clusters up to 100 nodes with 30 pods per node and 1-2 container per pod (as defined in the [1.0 roadmap](../../docs/roadmap.md#reliability-and-performance)).
+At v1.0, Kubernetes supports clusters up to 100 nodes with 30 pods per node and 1-2 container per pod.
```
Comment (lavalamp) [inline docs/admin/cluster-large.md]: containers

**Comment by lavalamp** [comment]
LGTM

File: docs/admin/cluster-large.md [hunk 2]
```diff
@@ -35,10 +35,12 @@ Documentation for other releases can be found at
...
 
 ## Setup
 
+Currently, a cluster is a set of nodes (VMs, machines), managed by a master, running a version of Kubernetes. This master is the cluster-level control-plane.
```
Comment (davidopp) [inline docs/admin/cluster-large.md]: Currently?

File: docs/api.md [hunk 3]
```diff
@@ -33,6 +33,8 @@ Documentation for other releases can be found at
 
 # The Kubernetes API
 
+For existing and future workloads, we want to provide a consistent, stable set of APIs, over which developers can build and extend Kubernetes. This includes input validation, a consistent API structure, clean semantics, and improved diagnosability of the system.
```
Comment (davidopp) [inline docs/api.md]: In the interest of keeping our documentation lightweight, I would suggest removing this. Maybe we can have a general philosophy doc somewhere (we talked about doing that at one point, I think).

File: docs/user-guide/overview.md [hunk 4]
```diff
@@ -33,7 +33,9 @@ Documentation for other releases can be found at
 
 # Kubernetes Overview
 
-Kubernetes is an open-source system for managing containerized applications across multiple hosts in a cluster. It provides mechanisms for application deployment, scheduling, updating, maintenance, and scaling. A key feature of Kubernetes is that it actively manages the containers to ensure that the state of the cluster continually matches the user's intentions.
+Kubernetes is an open-source system for managing containerized applications across multiple hosts in a cluster. For applications / micro-services that run on Kubernetes, we want deployments to be easy but powerful.
```
Comment (davidopp) [inline docs/user-guide/overview.md]: I'm not sure it's the deployments you want to be easy and powerful. Maybe "Kubernetes is intended to make deploying containerized/microservice-based applications easy but powerful."

**Comment by bgrant0607** [comment]
Addressed feedback. PTAL.

**Comment by bgrant0607** [comment]
Travis passed.

**Comment by davidopp** [comment]
Looking...

**Comment by davidopp** [comment]
LGTM, merging

**Pull request merged**